packages/grafana-data/src/types/annotations.ts

```diff
@@ -78,7 +78,7 @@ export interface AnnotationSupport<TQuery extends DataQuery = DataQuery, TAnno =
   /**
    * When the standard frame > event processing is insufficient, this allows explicit control of the mappings
    */
-  processEvents?(anno: TAnno, data: DataFrame): AnnotationEvent[] | undefined;
+  processEvents?(anno: TAnno, data: DataFrame[]): AnnotationEvent[] | undefined;
```

Comment:
This API is new in 7.2, so I think we should update before it gets released :grimacing: